docs/connector-development/connector-builder-ui/incremental-sync.md

```diff
@@ -0,0 +1,133 @@
+* "Cursor field" is set to `webPublicationDate`
+* "Datetime format" is set to `%Y-%m-%dT%H:%M:%SZ`
+* "Cursor granularity is set to `PT1S` as this API can handle date/time values on the second level
+* "Start datetime" is set to "config value"
+* "End datetime" is set to "now" to fetch all articles up to the current date
+* "Inject start time into outgoing HTTP request" is set to `request_parameter` with "Field" set to `from-date`
+* "Inject end time into outgoing HTTP request" is set to `request_parameter` with "Field" set to `to-date`
+
+This API orders records by default from new to old, which is not optimal for a reliable sync as the last encountered cursor value will be the most recent date even if some older records did not get synced (for example if a sync fails halfway through). It's better to start with the oldest records and work your way up to make sure that all older records are synced already once a certain date is encountered on a record. In this case the API can be configured to behave like this by setting an additional parameter:
+* At the bottom of the stream configuration page, add a new "Request parameter"
+* Set the key to `order-by`
+* Set the value to `oldest`
+
+Setting the start date in the "Testing values" to a week in the past (`2023-04-09T00:00:00Z` at the time of writing) results in the following request:
+```
+curl 'https://content.guardianapis.com/search?order-by=oldest&from-date=2023-04-09T00:00:00Z&to-date=2023-04-15T10:18:08Z'
+```
+
+The last encountered date will be saved as part of the connection - when the next sync is running, it picks up from the last record. Let's assume the last ecountered article looked like this:
+```
+{
+  "id": "business/live/2023/apr/15/uk-bosses-more-optimistic-energy-prices-fall-ai-spending-boom-economics-business-live",
+  "type": "liveblog",
+  "sectionId": "business",
+  "sectionName": "Business",
+  "webPublicationDate": "2023-04-15T07:30:58Z",
+}
+```
+
+Then when a sync is triggered for the same connection the next day, the following request is made:
+```
+curl 'https://content.guardianapis.com/search?order-by=oldest&from-date=2023-04-15T07:30:58Z&to-date=2023-04-16T10:18:08Z'
+```
+
+The `from-date` is set to the cutoff date of articles synced already and the `to-date` is set to the new "now".
+
+## Advanced settings
+
+The description above is sufficient for a lot of APIs. However there are some more subtle configurations which sometimes become relevant. 
+
+### Step
+
+When incremental syncs are enabled, the connector is not fetching all records since the cutoff date at once - instead it's splitting up the time range between the cutoff date and the desired end date into intervals based on the "Step" configuration (by default it's set to 1 month)
+
+For example if the "Step" is set to 10 days (`P10D`) for the Guardian articles stream described above and a longer time range, then the following requests will be performed:
```

Comment:
Can we point to some docs or an explanation of what the values are? `P10D` is very cryptic